docs_src/getting-started/Ch-GettingStartedCDevelopers.md

```diff
@@ -20,17 +20,18 @@ guide](./Ch-GettingStartedDevelopers.md), to build EdgeX C services, you will ne
 -   libuuid
 -   hiredis
 
-You can install these on Ubuntu by running:
+You can install these on Ubuntu (20.04 LTS) by running:
 ``` bash
 sudo apt-get install libcurl4-openssl-dev libmicrohttpd-dev libyaml-dev libcbor-dev libpaho-mqtt uuid-dev libhiredis-dev
 ```
+Some of these supporting packages have dependencies of their own, which will be automatically installed when using package managers such as `apt`, `yum` etc.
```

Comment:
'apt' => 'apt-get' like change below?

Reply:
I think apt is the name of the package management system and `apt-get` is one of the commands it provides. Maybe change `apt`, `yum` to _Apt_, _Yum_ ?

Reply:
Ahh ok. I am good with that.